launch_ros/launch_ros/utilities/normalize_parameters.py

```diff
@@ -74,7 +76,7 @@ def _normalize_parameter_array_value(value: SomeParameterValue) -> ParameterValu
         # all were floats or ints, so return floats
         make_mypy_happy_float = cast(List[Union[int, float]], value)
         return tuple(float(e) for e in make_mypy_happy_float)
-    elif Substitution in has_types and has_types.issubset({str, Substitution, tuple}):
+    elif Substitution in has_types and has_types.issubset({str, Substitution}):
```

Comment:
@ivanpauno why was `tuple` removed?

Reply:
It doesn't have sense, `normalize_to_list_of_substitutions` can't handle that and fails:
https://github.com/ros2/launch_ros/blob/78469a9122ad4ca8416bf3a6c34acecbfafb982f/launch_ros/launch_ros/utilities/normalize_parameters.py#L77-L79
I added a test case for that: https://github.com/ros2/launch_ros/blob/821395a5f3264d55d87eb407d65e917146de23dc/test_launch_ros/test/test_launch_ros/test_normalize_parameters.py#L95-L101.
Without removing `tuple`, it was failing.
The old test cases are passing with the removal.